docs/gyakorlat/ef/index.md

```diff
@@ -176,6 +176,84 @@ Debugger segítségével nézd meg, hogy milyen SQL utasítás generálódik: az
     }
     ```
 
+??? example "Megoldás method chain-nel"
```

Comment:
Azt kérném, hogy ez NE új szekcióban legyen, hanem az alternatív megoldások egymáshoz közel legyenek. Pl. a MongoDB-nél is van két szintaktika, amik közvetlenül egymás után szerepelnek:

![image](https://user-images.githubusercontent.com/11335181/144982579-f021024c-e509-4252-a2f8-2a37af8e185d.png)